tests/group_vars/vagrant.yml

```diff
@@ -416,7 +417,7 @@ s3:
   chunked_encoding: true
 
 schemaspy:
-  enabled: true
+  enabled: false
```

Comment:
This seems unrelated but harmless 😄

Reply:
turning off something I accidentally left on =)